vaadin-ordered-layout-flow-demo/src/main/java/com/vaadin/flow/component/orderedlayout/demo/OrderedLayoutView.java

```diff
@@ -354,23 +338,15 @@ private void createVerticalLayoutWithJustifyContent() {
                 "Component 2<br>With long text");
         component3.getElement().getStyle().set("fontSize", "9px");
 
-        Div buttons = new Div();
-        buttons.add(createSpacingButton(layout, "justify-content-start-button",
-                FlexComponent.JustifyContentMode.START));
-        buttons.add(createSpacingButton(layout, "justify-content-end-button",
-                FlexComponent.JustifyContentMode.END));
-        buttons.add(
-                createSpacingButton(layout, "justify-content-between-button",
-                        FlexComponent.JustifyContentMode.BETWEEN));
-        buttons.add(createSpacingButton(layout, "justify-content-around-button",
-                FlexComponent.JustifyContentMode.AROUND));
-        buttons.add(createSpacingButton(layout, "justify-content-evenly-button",
-                FlexComponent.JustifyContentMode.EVENLY));
-
+        RadioButtonGroup<FlexComponent.JustifyContentMode> contentMode = new RadioButtonGroup<>();
+        contentMode.setItems(FlexComponent.JustifyContentMode.values());
+        contentMode.setId("vertical-layout-justify-content-radio-button");
+        contentMode.addValueChangeListener(event -> layout.setJustifyContentMode(event.getValue()));
+        contentMode.setValue(FlexComponent.JustifyContentMode.BETWEEN);
         layout.setId("layout-with-justify-content");
```

Comment:
This demo doesn't work well. Because the layout should have a fixed height e.g. 300px.

Reply:
Indeed! Fixed